bridge.js

```diff
@@ -13,23 +12,11 @@ const hubName = process.env.TTN_AZURE_HUBNAME;
 const keyName = process.env.TTN_AZURE_KEYNAME;
 const key = process.env.TTN_AZURE_KEY;
 
-var options;
-
-try {
-  const mqttCertPath = process.env.TTN_MQTT_CERT;
-  if (!mqttCertPath) {
-    throw new Error('Could not find the ca certificate');
-  }
-  options = {
-    protocol: 'mqtts',
-    ca: [ fs.readFileSync(mqttCertPath) ],
-  };
-}
-catch(err) {
-  options = {};
-  console.warn('Could not connect using TLS %s', err);
-  console.warn("Fallback to a non-secure connection");
-}
+const mqttCertPath = process.env.TTN_MQTT_CERT || '/etc/ttn/mqtt-ca.pem'
+var options = {
+  protocol: 'mqtts',
+  ca: [ fs.readFileSync(mqttCertPath) ],
```

Comment:
It can be an array or a string. `ca<string> | <string[]> | <Buffer> | <Buffer[]>`